mautic/mautic
# [Bug] Contact limiter threads (2.14)

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Yes
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
A little mistake here. ThreadId gets passed as a boolean, meaning the multi-threaded approach ends up not working as intended. Simple fix.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. run `console mautic:campaigns:trigger --thread-id 1 --max-threads 2`
2. run `console mautic:campaigns:trigger --thread-id 2 --max-threads 2`
3. Both are consuming even leads.

#### Steps to test this PR:
1. run `console mautic:campaigns:trigger --thread-id 1 --max-threads 2`
2. run `console mautic:campaigns:trigger --thread-id 2 --max-threads 2`
3. In debug logs you should be able to see both of these handling leads that are odd and even respectively.
